packages/flutter/test/cupertino/route_test.dart

```diff
@@ -502,6 +502,165 @@ void main() {
     expect(tester.getTopLeft(find.byType(Placeholder)).dy, closeTo(600.0, 0.1));
   });
 
+  testWidgets('Bottom route has parallax when non fullscreen route is pushed on top', (WidgetTester tester) async {
```

Comment:
fullscreen dialog route has parallax exit animation when non fullscreen route is pushed on top

Reply:
This test is not targeting full screen specifically, also its not only the exit animation that has parallax
So maybe: CupertinoPageRoute has parallax animation when non fullscreenDialog route is pushed on top